tasks/images.yml

```diff
@@ -35,6 +35,8 @@
     state: absent
   when: item.force_rebuild | default(os_images_force_rebuild) | bool
   with_items: "{{ os_images_list }}"
```

Comment:
Another solution I've used is to loop over the list indices, then index into the list. You could also iterate over the names, then search the list to get back to the item. That's a bit more involved, but avoids the loop control.